### PR TITLE
Create a ssh-forward image for supporting local/remote port forwarding and …

### PR DESCRIPTION
**Pull request by andytson-inviqa**
…SOCKS proxy

This is multi-purpose for example to solve:

* Seeing behat tests running in a local browser
* Using PHP XDebug remote debugging in an IDE
* SOCKS proxying into Docker for Mac for direct browsing unexposed http/https

### CODE REVIEW

File: ssh-forward/usr/local/share/container/baseimage-20.sh [hunk 1]
```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+do_ssh_forward_set_credentials() {
+  if [ -n "${SSH_FORWARD_PASSWORD}" ]; then
+    echo "forward:${SSH_FORWARD_PASSWORD} | chpasswd
```
Comment (kierenevans) [inline ssh-forward/usr/local/share/container/baseimage-20.sh]: Missing quote mark

Comment (andytson-inviqa) [inline ssh-forward/usr/local/share/container/baseimage-20.sh]: fixed

**Comment by kierenevans** [comment]
Do we need fail2ban in here as well?

**Comment by andytson-inviqa** [comment]
I would suggest the port is never exposed publicly

**Pull request merged**